airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
Yes, but hidden here doesn't mean it's actually hidden, it means a potentially hidden variable. Only encrypted vars have a property accessor that will fail, and that is the only use case right now.